ajax/libs/e-modal/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "eModal",
```

Comment:
I think we better follow original name: `emodal` https://github.com/saribe/eModal/blob/master/package.json#L35